test/runtests.jl

```diff
@@ -103,10 +102,14 @@ if !is_APPVEYOR && GROUP == "ODEInterfaceRegression"
   @time @safetestset "ODEInterface Regression Tests" begin include("odeinterface/odeinterface_regression.jl") end
 end
 
+if !is_APPVEYOR && GROUP == "Multithreading"
+  @time @safetestset "Extrapolation Tests" begin include("algconvergence/ode_extrapolation_tests.jl") end
```

Comment:
@ChrisRackauckas Does file name here needs to be modified to `multithreading/ode_extrapolation_tests.jl`